Libraries/Components/AccessibilityInfo/AccessibilityInfo.windows.js

```diff
@@ -11,32 +11,55 @@
  */
 'use strict';
 
-type ChangeEventName = $Enum<{
-  change: string,
-}>;
+var NativeModules = require('NativeModules');
+var RCTDeviceEventEmitter = require('RCTDeviceEventEmitter');
+
+var RCTAccessibilityInfo = NativeModules.AccessibilityInfo;
+var HIGH_CONTRAST_EVENT = 'highContrastDidChange';
+var _subscriptions = new Map();
 
-var warning = require('fbjs/lib/warning');
+type AccessibilityEventName = $Enum<{
+  [HIGH_CONTRAST_EVENT]: string,
+}>;
 
 var AccessibilityInfo = {
 
-  fetch: function(): Promise<*> {
+  initialHighContrast: RCTAccessibilityInfo.initialHighContrast,
+  
+  fetch: function(): Promise {
     return new Promise((resolve, reject) => {
-      reject('AccessibilityInfo is not supported on this platform.');
+      resolve(false);
```

Comment:
>resolve(false); [](start = 6, length = 15)

Can you put a comment about how UWP doesn't provide anyway to query whether or not a screen reader is being used?

Reply:
Done